__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require('../app');
+const connection = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    if (connection) connection.end();
+})
+
+describe('GET /api/topics', () => {
+    it('status 200, responds with an array of topic objects,  each of which should have the following properties: slug, description.', () => {
+        return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body}) => {
+                const {topics} = body;
+                const {topicData} = testData;
+
+                expect(topics).toBeArray();
+                expect(topics.length).toBe(topicData.length);
+
+                topics.forEach(topic => {
+                    expect(topic).toHaveProperty('slug', expect.any(String));
+                    expect(topic).toHaveProperty('description', expect.any(String));
+                })
+            })
+    })
+})
```

Comment:
Need to add an additional Describe block for generic error handling - i.e. you can make a test for 404 - invalid paths that will cover all of the end points (don't write a test for 404's for all of your end points, you only need one).